docs/resources/aviatrix_controller_access_allow_list_config.md

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Settings"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_controller_access_allow_list_config"
+description: |-
+  Creates and manages an Aviatrix Controller's Access Allow List  Config
```

Comment:
One redundant between List and Config

Reply:
There is an extra space between List and Config